pkgs/applications/audio/mamba/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, xorg, cairo, libjack2, pcre, libXdmcp, libsigcxx, liblo, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "Mamba";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "brummer10";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rjy7l9sx8xvk64s29g0jrrirm5h2qg0pdvvz4i1kkbs7nasnjm5";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    xorg.libX11 xorg.libpthreadstubs cairo libjack2 pcre libXdmcp libsigcxx liblo
```

Comment:
Can you format your contributions with nixpkgs-fmt?
(i.e. `nix run -f channel:nixos-unstable nixpkgs-fmt -c nixpkgs-fmt pkgs/applications/audio/mamba/default.nix`)